influxdb/client.py

```diff
@@ -61,6 +61,11 @@ class InfluxDBClient(object):
     :type proxies: dict
     :param path: path of InfluxDB on the server to connect, defaults to ''
     :type path: str
+    :param cert: Path to client certificate to use for mutual TLS
```

Comment:
I would suggest to be more precise about the fact that this path is pointing to a file containing *both* the client certificate and its private key.